CNAME

```diff
@@ -0,0 +1 @@
+fedspendingtransparency.18f.us
```

Comment:
Actually we can remove this file from the PR. This is to set the domain name to point the dev site at our fork, but we don't want this to be merged upstream. It will screw with the hosting.